### PR TITLE
Masonry Projects Page

### PR DESCRIPTION
**Pull request by ijemmao**
### Description
* Implemented masonry isotopic layout
* Using live Project data
* I hardcoded in the images just for presentational purposes

### Screenshots
Four different block sizes:
* 2 x 1 - `medium-width`
* 3 x 1 - `long-width`
* 1 x 2 - `long-height`
* 1 x 1 - `block`

screen shot 2018-10-06 at 6 36 43 pm
screen shot 2018-10-06 at 6 37 14 pm

### CODE REVIEW

File: src/components/cards/project_card.js [hunk 1]
```diff
@@ -3,19 +3,38 @@ import ProjectUserList from '../project_card_user_list';
...
-      <div className="project_card" style={{ textDecoration: 'none' }}>
+      <div className={`project_card grid-item ${this.props.blockSize}`} style={{
+        backgroundRepeat: 'no-repeat', backgroundSize: 'cover',
+        backgroundImage: "url('https://news.dartmouth.edu/sites/dart_news.prod/files/styles/slide/public/news/images/pulse_app_028-810.jpg?itok=nr8SYcKN')", textDecoration: 'none', width: this.props.width, height: this.props.height }}>
         <Link to={`/projects/${this.props.project.id}`} className="link">
+          <div className="mask"/>
           <div id="top">
-            <h4>{this.props.project.title}</h4>
-            <p>{this.props.project.techStack}</p>
+            {/* Project image must have a 1:1 ratio */}
```
Comment (randaline11) [inline src/components/cards/project_card.js:27]: Can we put this in documentation somewhere if it isn't already?

File: src/components/cards/project_card.js [hunk 2]
```diff
@@ -3,19 +3,38 @@ import ProjectUserList from '../project_card_user_list';
...
+            <img className="project-image" src="https://collegepulse.com/img/logo.png" />
+            <h4>{this.formatTitle(this.props.project.title)}</h4>
+            {/* <p>{this.props.project.techStack}</p> */}
+          </div>
+          <div id="bottom">
+            <h3>{this.props.project.shortDescription}</h3>
+            {/* <h3>Gauging public opinion on college campuses</h3> */}
+            {/* <ProjectUserList members={this.props.project.members} /> */}
```
Comment (randaline11) [inline src/components/cards/project_card.js:34]: why are these 2 lines commented out? Like are they examples?

File: src/pages/projects_page.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ProjectCard from '../components/cards/project_card';
```
Comment (randaline11) [inline src/pages/projects_page.js]: I think technically absolute imports should come before relative imports but that's pretty minor

File: src/pages/projects_page.js [hunk 4]
```diff
@@ -26,28 +39,28 @@ class Projects extends React.Component {
       });
     });
   }
-
   render() {
+    this.initializeGrid();
+    // const displayedProjects = ['med-width', 'long-height', 'block', 'block', 'long-width'].map((item) => {
```
Comment (randaline11) [inline src/pages/projects_page.js:44]: why is this commented out?

File: src/stylesheets/components/_project_card.scss [hunk 5]
```diff
@@ -1,20 +1,48 @@
 .project_card {
+  font-family: $project-header-font;
+  .mask {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    border-radius: 15px;
+    background-color: rgba(0, 0, 0, 0.3);
```
Comment (randaline11) [inline src/stylesheets/components/_project_card.scss]: If we're using this color ever again, maybe we should make it a variable :)

File: src/stylesheets/components/_project_card.scss [hunk 6]
```diff
@@ -38,3 +66,54 @@
     }
   }
 }
+
+.grid-item {
+    float: left;
+    position: relative;
+    border-radius: 15px;
+    box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.1);
```
Comment (randaline11) [inline src/stylesheets/components/_project_card.scss]: same here... maybe we can pre-define all of these shadows?

**Review by randaline11 — COMMENTED** [review]
I love the animation! super clean. The text looks a little too big when it's 3 lines though.... some of my friends were pointing out the photo instead of the text. Ex. Terren's eyes were in between the lines, etc. There's only supposed to be one block correct? I'll try and add another project to the DB tonight so we can try the other rectangle sizes.

File: src/pages/projects_page.js (hunk 4 above)
Comment (ijemmao) [reply src/pages/projects_page.js:44]: That's to generate all the different sized boxes in the example. Didn't want to delete it so I can easy access testing once this feature is finalized.

File: src/pages/projects_page.js (hunk 3 above)
Comment (ijemmao) [reply src/pages/projects_page.js]: That's a fair point!

File: src/components/cards/project_card.js (hunk 1 above)
Comment (ijemmao) [reply src/components/cards/project_card.js:27]: Do you know where I should place this information? I don't think I should put it in the `README.md` file.

File: src/components/cards/project_card.js (hunk 2 above)
Comment (ijemmao) [reply src/components/cards/project_card.js:34]: That's odd, both commented lines aren't showing on my local project. It seems like when I try to push it doesn't update the remote repo's verison😕

File: src/pages/projects_page.js (hunk 4 above)
Comment (randaline11) [reply src/pages/projects_page.js:44]: gotcha :)

File: src/components/cards/project_card.js (hunk 1 above)
Comment (randaline11) [reply src/components/cards/project_card.js:27]: can we keep it here, duplicate in readme for now, and once we have a better spot (ex. the handoff) we move it out of the readme? I'm just worried people might forget about it if it's only in the code

**Review by randaline11 — APPROVED** [review]

**Pull request merged**